src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import core.basesyntax.enums.FigureType;
+import java.util.Random;
+
+public class FigureSupplier {
```

Comment:
The `FigureSupplier` class should not contain an empty line at the beginning of its definition.